pages/account.js

```diff
@@ -24,8 +24,12 @@ class Account extends Component {
             email: '',
             newName: '',
             newEmail: '',
-            failedProfileUpdate: false,
             succeededProfileUpdate: false,
+            failedProfileUpdate: false,
+            oldPasswordInput: '',
+            newPasswordInput: '',
+            succeededPasswordUpdate: false,
+            failedPasswordUpdate: false,
```

Comment:
I wonder if we could handle these states more elegantly by having a single piece of state like:
```
passwordUpdateStatus: <takes values true, false, null>
```
and same for `profileUpdateStatus`

if null, means nothing has happened
if true means success
if false means failed

would avoid having like 4 pieces of state for one task